src/backend/marsha/core/utils/medialive_utils.py

```diff
@@ -279,10 +294,13 @@ def delete_aws_element_stack(video):
         - mediapackage channel and endpoints
     """
     # Mediapackage
-    # First delete mediapackage endpoint
+    # First delete mediapackage endpoints
```

Comment:
Why do not iterate over `video.live_info.get("mediapackage").get("endpoints")` ?

Reply:
I can't because I have to know which one to delete. But I can do a loop like this `for endpoint in ["hls", "dash"]:`

Reply:
Oh ok. I understood that you wanted to delete all endpoints contained in mediapackage at this step. So I though more flexible to iterate over this dictionnary.